hs_modelinstance/templates/pages/modelinstanceresource.html

```diff
@@ -29,11 +29,31 @@
                         <h4><strong>Executed By:</strong></h4>
                         <hr style="margin-top: 0px;margin-bottom: 2px">
                         <div class="row">
-                            <div class="col-sm-4"><strong>Model Program used for execution:</strong></div>
-                            <div class="col-sm-8">{{ executed_by.name }}</div>
+
+                        <div class="col-sm-4"><strong>Name</strong></div>
+                        <div class="col-sm-8">{{ executed_by.model_program_fk.title }}</div>
+
+                        <div class="col-sm-4"><strong>Version</strong></div>
+                        <div class="col-sm-8">
+                            {% if executed_by.model_program_fk.metadata.mpmetadata.first.software_version.strip %}
+                                {{executed_by.model_program_fk.metadata.mpmetadata.first.software_version}}
+                            {% else %}
+                                Not Provided
+
+                            {% endif %}
                         </div>
+
+                        <div class="col-sm-4"><strong>Resource URL</strong></div>
+
+
+                       <div class="col-sm-8"> <a id="permalink" href="http://{{ request.get_host }}/resource/{{ executed_by.model_program_fk.short_id }}/">http://{{ request.get_host }}/resource/{{ executed_by.model_program_fk.short_id }}</a></div>
```

Comment:
Remove the `http:` so that if the site changes to HTTPS this link is relative to the current schema